modules/help_editor/jsx/help_editor.js

```diff
@@ -0,0 +1,135 @@
+import FilterForm from 'FilterForm';
+import formatColumn from './columnFormatter';
+
+/**
+ * Help Editor Archive Page.
+ *
+ * Serves as an entry-point to the module, rendering the whole react
+ * component page on load.
+ *
+ * Renders Help Editor main page consisting of FilterTable and
+ * DataTable components.
+ *
+ * @author Alex Ilea
```

Comment:
Since you made this file, I think you can author it ;). Or you put can put `Loris Team`.